megamek/src/megamek/common/loaders/BLKWarshipFile.java

```diff
@@ -185,7 +185,7 @@ public Entity getEntity() throws EntityLoadingException {
 		if (dataFile.exists("overview")) {
 			a.getFluff().setOverview(dataFile.getDataAsString("overview")[0]);
 		}
-        // Grav Decks - two approaches
+		// Grav Decks - two approaches
```

Comment:
Tabs were introduced.

Reply:
Huh. I have my tabs set up to enter 4 spaces, and they work correctly on every line but this one - even when I remove it and re-enter.  Fixed, but that's weird.